src/client/core/extensions/utils.js

```diff
@@ -0,0 +1,65 @@
+import hammerhead from '../deps/hammerhead';
+
+
+export const CONTAINS_OWN_TEXT_METHOD_NAME = 'containsExcludeChildren';
+export const ATTR_REGEXP_METHOD_NAME       = 'attrRegExp';
+
+const TEXT_ELEMENT_TAGS_REGEXP      = /^i$|^b$|^big$|^small$|^em$|^strong$|^dfn$|^code$|^samp$|^kbd$|^var$|^cite$|^abbr$|^acronym$|^sub$|^sup$|span$|^bdo$|^address$|^div$|^a$|^object$|^p$|^h\d$|^pre$|^q$|^ins$|^del$|^dt$|^dd$|^li$|^label$|^option$|^textarea$|^fieldset$|^legend$|^button$|^caption$|^td$|^th$|^title$/;
```

Comment:
Use `RE` postfix for regexp vars naming:
`TEXT_ELEMENT_TAGS_RE`
`START_SUBSTR_RE`
...